src/profile-logic/marker-schema.js

```diff
@@ -394,8 +404,118 @@ export function formatFromMarkerSchema(
       return formatPercent(value);
     default:
       console.error(
-        `A marker schema of type "${markerType}" had an unknown format "${(format: empty)}"`
+        `A marker schema of type "${markerType}" had an unknown format ${JSON.stringify(
+          format
+        )}`
       );
       return value;
   }
 }
+
+export function isDOMRenderingFormat(format: MarkerFormatType): boolean {
+  return format === 'url' || typeof format === 'object' || format === 'list';
+}
+
+function _formatDOMFromMarkerSchema(
```

Comment:
bikeshedding nit: I'd call this `formatMarkupFromMarkerSchema` -- indeed DOM means something different, this is the tree that results from parsing markup, and this is also a specified API.